pkgs/tools/admin/ansible/default.nix

```diff
@@ -5,20 +5,7 @@ rec {
 
   ansible_2_12 = python3Packages.toPythonApplication python3Packages.ansible-core;
 
-  ansible_2_11 = python3Packages.toPythonApplication (python3Packages.ansible-core.overridePythonAttrs (old: rec {
-    pname = "ansible-core";
-    version = "2.11.6";
-
-    src = python3Packages.fetchPypi {
-      inherit pname version;
-      sha256 = "sha256-k9UCg8fFtHbev4PcCJs/Z5uTmouae11ijSjar7s9MDo=";
-    };
-  }));
-
-  ansible_2_10 = python3Packages.toPythonApplication python3Packages.ansible-base;
-
-  # End of support 2021/10/02, End of life 2021/12/31
-  ansible_2_9 = python3Packages.toPythonApplication python3Packages.ansible;
-
-  ansible_2_8 = throw "Ansible 2.8 went end of life on 2021/01/03 and has subsequently been dropped";
+  ansible_2_11 = throw "Ansible 2.11 goes end of life in 2022/11 and can't be supported throughout the 22.05 release cycle";
```

Comment:
I'm not overly familiar with python packaging, but can't we put these in `pkgs/top-level/python-aliases.nix` or even `pkgs/top-level/aliases.nix`? 

I think we have a process for removing things from `pkgs/top-level/aliases.nix` after a certain time now, but it (probably) doesn't apply to throws in random places in the codebase.

Reply:
Yeah, I think this whole file is a bit weird. Will update this in a bit.